content/references/protocol-reference/ledger-data/ledger-object-types/nftokenpage.md

```diff
@@ -11,7 +11,7 @@ status: not_enabled
 # NFTokenPage
 {% include '_snippets/nfts-disclaimer.md' %}
 
-The `NFTokenPage` object represents a collection of `NFToken` objects owned by the same account. An account can have multiple `NFTokenPage` ledger objects, which form a doubly-linked list (DLL).
+The `NFTokenPage` object represents a collection of `NFToken` objects owned by the same account. An account can have multiple `NFTokenPage` ledger objects, which form a doubly linked list (DLL).
```

Comment:
Since the abbreviation "DLL" isn't used again (and `.dll` has a different meaning in Windows files, so people could get led astray thinking about those), I suggest removing it.

```suggestion
The `NFTokenPage` object represents a collection of `NFToken` objects owned by the same account. An account can have multiple `NFTokenPage` ledger objects, which form a doubly linked list.
```